### PR TITLE
[Documentation] Fixes #1632: Updated release documentation

### LINKED ISSUE #1632: Update Release Documentation
**What would you like to be added**:
[The release doc](https://github.com/Seneca-CDOT/telescope/blob/master/docs/release.md) needs to be updated to reflect the new release automation workflow added in https://github.com/Seneca-CDOT/telescope/pull/1628.
**Why would you like this to be added**:
Since our release process changed, we need to get our docs up-to-date with it.

### PR DESCRIPTION
**Pull request by birtony**
## Issue This PR Addresses
Fixes #1632

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [x] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description
This PR updates our [release documentation](https://github.com/Seneca-CDOT/telescope/blob/master/docs/release.md) to account for the recent introduction of the automated [release workflow](https://github.com/Seneca-CDOT/telescope/blob/master/.github/workflows/release.yml) (https://github.com/Seneca-CDOT/telescope/pull/1628 and https://github.com/Seneca-CDOT/telescope/pull/1648).

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [x] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -39,13 +42,26 @@ To create a new release, follow these steps:
...
+
+To create a new release, follow these steps:
+
+1. Make sure your `master` branch is up-to-date, and you have the most recent git tags in your repo: `git pull upstream master --tags`.
+1. Run `npm outdated` to see a report of [outdated npm packages](https://docs.npmjs.com/cli-commands/outdated.html). Examine this list and decide whether we need to update
+   anything now, or before the next release. _Please file a new Issue to update
+   anything that is outdated_.
+1. Make sure your working tree is clean.
+1. Determine what the new version _string_ should be based on [semantic versioning](https://github.com/npm/node-semver#functions). The `newversion` argument should be a valid semver string. In our project, that would usually be `minor` (e.g. `1.5.0`, `1.6.0`, `1.7.0`) or `major` (e.g. `1.0.0`, `2.0.0`). See [npm-version docs](https://docs.npmjs.com/cli/v6/commands/npm-version) to learn more about what options are available for the new version string.
+1. Use `npm version <new-version-string> -m "Release message` to trigger the automated release workflow.
```
Comment (humphd) [inline docs/release.md:60]: Give an example of what it would look like exactly after this sentence.

File: docs/release.md [hunk 2]
```diff
@@ -39,13 +42,26 @@ To create a new release, follow these steps:
...
+
+`npm-version` will proceed to run tests locally. If successful, it will also bump the `version` in [package.json](https://github.com/Seneca-CDOT/telescope/blob/master/package.json), create a new [`git tag`](https://git-scm.com/book/en/v2/Git-Basics-Tagging) and push both the code and the tags to `upstream master`.
```
Comment (humphd) [inline docs/release.md:62]: Add a note about how you must be an admin and must have an upstream pointing to the main Seneca CDOT telescope repo

**Review by humphd — CHANGES_REQUESTED** [review]

File: docs/release.md (hunk 2 above)
Comment (birtony) [reply docs/release.md]: Updated the whole, added a note about this [here](https://github.com/Seneca-CDOT/telescope/pull/1650/files#diff-87479ac9ea16cec5ceafce00b3b049d7d5920cb663afae42901f20e7c4e6705dR11-R12)

File: docs/release.md (hunk 1 above)
Comment (birtony) [reply docs/release.md]: Added examples [here](https://github.com/Seneca-CDOT/telescope/pull/1650/files#diff-87479ac9ea16cec5ceafce00b3b049d7d5920cb663afae42901f20e7c4e6705dR36)

**Review by PedroFonsecaDEV — DISMISSED** [review]

**Review by PedroFonsecaDEV — DISMISSED** [review]

**Review by PedroFonsecaDEV — APPROVED** [review]

**Review by HyperTHD — APPROVED** [review]
LGTM

**Pull request merged**